webapp/src/main/java/com/box/l10n/mojito/service/ai/translation/AITranslationConfiguration.java

```diff
@@ -0,0 +1,85 @@
+package com.box.l10n.mojito.service.ai.translation;
+
+import com.google.common.collect.Maps;
+import java.time.Duration;
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties("l10n.ai.translation")
+public class AITranslationConfiguration {
+
+  private boolean enabled = false;
+  private int batchSize = 1000;
+
+  /**
+   * Duration after which a pending MT is considered expired and will not be processed in AI
+   * translation (as it will be eligible for third party syncs once the entity is older than the
+   * expiry period).
+   *
+   * <p>If the pending MT is expired, it will be deleted which will remove it from AI translation
+   * flow.
+   */
+  private Duration expiryDuration = Duration.ofHours(3);
+
+  private String cron = "0 0/10 * * * ?";
+
+  private Map<String, RepositorySettings> repositorySettings = Maps.newHashMap();
+
+  public static class RepositorySettings {
+    private boolean reuseSourceOnLanguageMatch = false;
+
+    public Boolean isReuseSourceOnLanguageMatch() {
```

Comment:
reactivating here: could we document in code what this property does, when it applies/for what type of scenario/when it should be used?